src/app/core/services/common/mapping.service.ts

```
@@ -168,15 +168,20 @@ export class MappingService {
     const workspaceId = this.workspaceService.getWorkspaceId();
-    const params: {limit: number, offset: number, attribute_type: string, active?: boolean, value__icontains?: string, value?: string, display_name__in?: string, detail__account_type__in?: string[], detail__category__in?: string[], destination_id__in?: string[]} = {
+    const params: {limit: number, offset: number, attribute_type?: string | string[], attribute_type__in?: string[], active?: boolean, value__icontains?: string, value?: string, display_name__in?: string, detail__account_type__in?: string[], detail__category__in?: string[], destination_id__in?: string[]} = {
       limit: 100,
       offset: 0,
       attribute_type: attributeType,
       active: true
     };
 
+    if (attributeType && Array.isArray(attributeType)) {
+      params.attribute_type__in = attributeType;
+      delete params.attribute_type;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)